src/builder.py

```diff
@@ -241,24 +261,42 @@ def from_armature(armature_object):
             # For now we can just use the first node!
             for time in keyframe_timings[model.nodes[0].name]['rotation_quaternion']:
                 # Expand our time
-                scaled_time = time * (1.0/0.025)
+                scaled_time = time * (1.0/get_framerate())
+                bpy.context.scene.frame_set(time, subframe=time-floor(time))
```

Comment:
```suggestion
                subframe_time = time - floor(time)
                bpy.context.scene.frame_set(time, subframe=subframe_time)
```
Makes it look a little cleaner